NEWS.md

```diff
@@ -87,6 +87,15 @@
   this doesn't work properly as it might result in leftover files in the install directory from a previous
   installation, so we have to empty the directory before reusing it, invalidating the caching, so the option
   was removed.
+- Build scripts are now executed on the host. See the `SCRIPTS` section
+  in the manual for more information. To run a script in the image, put
+  the following snippet at the top of the script:
+
+  ```sh
+  if [ ! "$container" = "mkosi" ]; then
+    exec chroot $SCRIPT "$@"
```

Comment:
I'm with @keszybz on calling this something else, e.g. as he proposed `_chroot` or maybe `mkosi-chroot` to be clear this is us.